Composer/packages/client/src/components/ProjectTree/ProjectTree.tsx

```diff
@@ -358,6 +366,20 @@ export const ProjectTree: React.FC<Props> = ({
           onDialogCreateTrigger(skillId, dialog.id);
         },
       },
+      {
+        label: formatMessage('Create KB from scratch'),
```

Comment:
are these labels passed by UX?

Reply:
UI has been updated with the latest copy now